kafkastats/src/main/java/com/pinterest/doctorkafka/stats/KafkaStatsMain.java

```diff
@@ -122,14 +118,16 @@ public static void main(String[] args) throws Exception {
     String kafkaConfigPath = commandLine.getOptionValue(KAFKA_CONFIG);
     long uptimeInSeconds = Long.parseLong(commandLine.getOptionValue(UPTIME_IN_SECONDS));
     long pollingInterval = Long.parseLong(commandLine.getOptionValue(POLLING_INTERVAL));
+    String primaryNetworkInterfaceName = commandLine.getOptionValue(PRIMARY_INTERFACE_NAME, "eth0");
```

Comment:
declare "eth0" as a constant like `DEFAULT_PRIMARY_INTERFACE_NAME` at the beginning of the class?

Reply:
done